ta/os_test/include/ta_os_test.h

```diff
@@ -49,5 +49,6 @@
 #define TA_OS_TEST_CMD_CALL_LIB_DL          16
 #define TA_OS_TEST_CMD_CALL_LIB_DL_PANIC    17
 #define TA_OS_TEST_CMD_GET_GLOBAL_VAR       18
+#define TA_OS_TEST_CMD_PARAMS_TMPMEM        19
```

Comment:
Suggest to rename `TA_OS_TEST_CMD_NULL_MEMREF_PARAMS`.

Reply:
ok